corpus/config/models.py

```diff
@@ -0,0 +1,65 @@
+from django.db import models
+
+
+class Society(models.Model):
+    """
+    Society Model.
+    Defines all societies that are part of IEEE NITK SB.
+    """
+
+    IEEE_SOCIETIES = [
```

Comment:
Is this an exhaustive list of all IEEE societies? Because what if there is a new society that is not part of this list? Is that ever possible?

Reply:
Yes, this is an exhaustive list of all IEEE societies, found [here](https://www.ieee.org/communities/societies/index.html).

It may be possible for new societies to be created by IEEE (although I don't know when the last one was created). However, I didn't find an API that distributed IEEE Society names. If there is a new society created by IEEE, we'll have to manually add onto this list.